.env.sample

```diff
@@ -7,4 +7,6 @@ ADMIN_PASSWORD=password
 WEBHOOKS_CACHE_TTL=300000
 NODE_ENV=dev
 SSE_AUTH_TOKEN=aW5mcmFAc2FmZS5nbG9iYWw6YWJjMTIz
+DB_SSL_ENABLE=false
```

Comment:
I would use 0 or 1 instead of a string like `false`.

Also we should use the same naming, as the original variable is `DATABASE_URL` this should be `DATABASE_SSL_ENABLED`, same for the CA path